app/src/main/java/com/sun/cocktaildb/screen/authenticate/login/LoginContract.kt

```diff
@@ -0,0 +1,19 @@
+package com.sun.cocktaildb.screen.authenticate.login
+
+interface LoginContract {
+    interface View {
+        fun showLoading()
+        fun hideLoading()
+        fun showError(message: String)
+        fun showSuccess()
+        fun navigateToMain()
+        fun getEmail(): String
+        fun getPassword(): String
+        fun clearInputs()
+    }
+
+    interface Presenter : com.sun.cocktaildb.utils.base.BasePresenter<View> {
+        fun login()
+        fun validateInputs(): Boolean
+    }
+} 
```

Comment:
```suggestion
} 

```
add blank line end of file